<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mainTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="entriesTab">
     <attribute name="title">
      <string>Entries</string>
     </attribute>
     <widget class="QPushButton" name="btnAddEntry">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelete">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableEntries">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>721</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>291</width>
        <height>133</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblFood">
           <property name="text">
            <string>Food</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbFood"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblDate">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="editDate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lblVolume">
           <property name="text">
            <string>Volume (ml)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinVolume"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblMass">
           <property name="text">
            <string>Mass (g)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinMass"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="foodTab">
     <attribute name="title">
      <string>Foods</string>
     </attribute>
     <widget class="QGroupBox" name="macronutrientsGroup">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Macronutrients</string>
      </property>
      <widget class="QLabel" name="lblPer">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblProteins">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Proteins</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblCarbs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Carbohydrates</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinCarbsPerc">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinProtPerc">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinSatFatPerc">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>42</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinUnsatFatPerc">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lblUnsatFat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Unsaturated fats</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblSatFat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Saturated fats</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="lblDensity">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Density</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinDensity">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblFoodName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFoodName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="FoodTable" name="tableFood">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>661</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddFood">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="graphTab">
     <attribute name="title">
      <string>Graphs</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FoodTable</class>
   <extends>QTableWidget</extends>
   <header>foodtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
